### ISSUE TITLE
Make Tugboat PR previews expire after a certain time

### ISSUE THREAD

**Issue by ghost** [backdrop/backdrop-issues#4739]
We now have Tugboat.qa working to provide preview sandboxes for all PRs against core. But since we have a 40GB limit on our Tugboat account, we need to be careful about how many PR sandboxes we are running.

One solution to that is to automatically expire (i.e. delete) PR sandboxes after a certain time. @quicksketch suggested initially setting this to 6 months. Then we can adjust as needed after we seehow that goes.

So this issue is to make that expiry happen. I talked to the Tugboat guys on Slack and they said it's not possible to do this through the UI, but it can be done via their API. I just need to work out how to integrate our Tugboat PRs with their API...

**Comment by BWPanda** [comment]
We could add the expiry date to the comment posted in the PR by dragonbot. E.g.:

> Tugboat has finished building a preview for this pull request!
>
> Website: https://pr3252-vwbemqrd2ks16hadmqtvedv1vjkthy2u.tugboat.qa/
> Username: admin
> Password: e1fd580932d6
>
> This preview will expire on Thursday the 6th of May, 2021.

**Comment by BWPanda** [comment]
Here's a PR, though I'm not sure it works... Don't know exactly how to test this either. Just putting it here so it's not lost, and so maybe others can help with it if possible...

https://github.com/backdrop/backdrop/pull/3824

**Comment by klonos** [comment]
Code looks good @BWPanda thanks 🙏🏼 

Pinging some others that come to mind for a code review and pair of 👀  @backdrop/core-committers @indigoxela

**Comment by BWPanda** [comment]
I tested this in my own repo, and I found the reason why it wasn't working (an issue with the Tugboat token set in Tugboat itself). That's been fixed, and I've updated to PR to expire previews after 2 months (which is closer to when we're hitting our quota currently (we can easily change this if/when we get more quota)), and to add a note to the PR comment about when the preview will expire:

![image](https://user-images.githubusercontent.com/2385329/161885472-e6f7c1b2-a020-4349-8ea3-4b53812a2dbc.png)

Considering how hard it is for others to test this, I think I'll just go ahead and merge it. We can tweak it as we go if necessary.
